src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -16,117 +17,41 @@
 
 namespace Mono.Linker.Dataflow
 {
-	sealed class ReflectionMethodBodyScanner : MethodBodyScanner
+	class ReflectionHandler
```

Comment:
I think a better name would be `TrimAnalysisHandler`. Two reasons:
* "Reflection" is not precise anymore - what this code does is more than that - it handles annotations and RUC and so on. In the analyzer and shared code we refer to this functionality as "trim analysis".
* This is the linker counterpart of `TrimAnalysisVisitor` from the analyzer - they perform basically the same services.

@sbomer, @tlakollo  - opinions?